### ISSUE TITLE
When dialog is open from dragging balloon, balloon does not receive focus on close

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#142]
New issue due to the change in balloon interaction.  The balloon's accessible content changes from a button to a div depending on its dragging state.  If the dialog is open from the balloon while the balloon is being dragged, focus moves from balloon to dialog.  When this happens, the balloon's accessible content changes back to the button and the div no longer exists in the document.  Since the previous active element does not exist, the balloon cannot automatically receive focus.  Pressing 'tab' again will set the focus to the balloon as expected.

**Comment by jessegreenberg** [comment]
This week, @terracoda suggested the possibility of keeping the balloon in its 'dragged' state when the keyboard help dialog is opened.  This is still under consideration, but that would solve this issue because the div representation of the balloon would still exist in the DOM.

**Comment by terracoda** [comment]
@jessegreenberg Interesting development! Maybe my intuition was correct :-)

**Comment by jessegreenberg** [comment]
Indeed, I think it would be a good option!  If we decide not to do this though, we can come up with another solution because I imagine that this will be a general problem that we encounter again in the future.

**Comment by terracoda** [comment]
@jessegreenberg BTW, I see the Hotkeys and Keyboard Help dialog is now available from anywhere. This is great! And would have been very useful during interviews :-) With this kind of On-Demand Help, we can add a reference to how this works in the Scene Summary like I had originally.

**Comment by jessegreenberg** [comment]
@terracoda unfortunately the Hotkeys and Keyboard Help dialog is only available everywhere because you were testing without a screen reader.  When a screen reader is in use, the keyboard help will only be available when the balloon has been grabbed, since the grabbed balloon has the 'application' role.

**Comment by jessegreenberg** [comment]
This was fixed and balloon receives focus when keyboard help dialog is opened from dragging balloon.  However, the recommendation from @terracoda could still be done at any time!  That will be tracked in another issue, so closing this one.
